contributors/chairs-and-techleads/README.md

```diff
@@ -0,0 +1,29 @@
+future home for chairs and tech leads resources and materials for onboarding,
+offboarding, operations, and more. files like zoom guidelines will still remain
+in their current home but included in documentation here in the form of canonical
+links.   
+
+
+Resources for community group leads:    
+
+operations  
+TODO  
+
+governance  
+TODO  
+
+mentoring, succession planning, and staffing  
+TODO  
+
+code of conduct  
+TODO  
+
+
+Communication and feedback loops:  
+Need to Know email archive [link]  
```

Comment:
Need to know: https://docs.google.com/document/d/1ivmV-ouim7YcTnmv21m0pP6prmj-FFZxcRBuWbT706c/edit